gen_conda_requirements.py

```diff
@@ -0,0 +1,90 @@
+# (C) British Crown Copyright 2017, Met Office
+#
+# This file is part of Iris.
+#
+# Iris is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Iris is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import os.path
+
+
+REQS_DIR = os.path.dirname(__file__)
+CONDA_PATTERN = '#conda:'
+
+
+def read_conda_reqs(fname, options):
+    lines = []
+    with open(fname, 'r') as fh:
+        for line in fh:
+            line = line.strip()
+            if '#conda:' in line:
```

Comment:
I know you didn't write this but y "u" no `CONDA_PATTERN`

Reply:
Sorry. I'm not sure I follow your comment.
(And yes, I did not write it, shameless stolen from `iris` :smile:)

Reply:
The following line gets the index/length of `CONDA_PATTERN`, and up above `CONDA_PATTERN = '#conda:'`, but for some reason it re-uses the bare string on this line.

Reply:
Ah OK. I did not see the line below until I click `/files` on GitHub.
I can fix that, this script was probably not designed to last. I would love to get rid of it as soon as conda start supporting the new req file standards.